_layouts/post.html

```diff
@@ -1,58 +1,66 @@
-              {% endfor %}
-            {% endif %}
-          </div>
-          <div class="content">
-              {{ content }}
-          </div>
-
-    </div>
-    <div class="col-md-2">
-        <div class="card contact-us-card text-center">
-          <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-            <div class="card-body">
-              <p class="card-title">Want to learn more? Feel free to contact us for support</p>
-              <a href="#" class="btn btn-primary">Support</a>
-              <a href="#">Dismiss</a>
-            </div>
-          </div>
     </div>
+    <div class="col-md-7 guides-section-white">
+      <ol class="breadcrumb text-center">
+        {% assign crumbs = page.url | split: '/' %}
+        <a href="/guides" ga-event-category="guide-{{ post.title | downcase}}" ga-event-action="deployment-guides">Dev Ops Guides</a> 
+        {% for crumb in crumbs offset: 2 %}
+          {% if forloop.last %}
+           <a href="#"> / {{ crumb | replace:'-',' ' | capitalize }}</a>
+          {% else %}
+            <a href="{{ crumb }}">/ {{ crumb | replace:'-',' ' | capitalize }}</a>
+          {% endif %}
+        {% endfor %}      
+      </ol>
+      <div class="post-title text-center">
+        <img class="img-center" src="{{ page.image }}" alt="post logo">
+        <label>{{ page.title }}</label>
+        <p>
+          {% assign words = content | number_of_words %}
+          <!-- https://help.medium.com/hc/en-us/articles/214991667-Read-time -->
+          {% if words < 265 %}
+            1 min read.
+          {% else %}
+            {{ words | divided_by:180 }} mins read.
+          {% endif %} 
+          Updated {{ page.date | date_to_string }}
+        </p>
+        {% if page.tags %}
+          {% for tag in page.tags %}
+            <span class="badge badge-pill badge-default">
+              {% capture tag-name %}{{tag | upcase }}{% endcapture %}
+              {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
+          {% endfor %}
+        {% endif %}
+      </div>
+      <div class="content">
+          {{ content }}
+      </div>
     </div>
+    <div class="col-sm-2-5">
+      <div class="card guides-cta-card text-center">
+        <img src="/assets/img/grunty-cta.png" alt="newsletter_image" class="img-center"><br>
+        <div class="card-body">
+          <p class="card-title">Hand off the gruntwork, we can deploy this service for you. </p>
+          <button type="button" class="btn deploy" ga-event-category="guide-cta" ga-event-action="guide-deploy-cta">Deploy for me</button>
+          <button type="button" class="btn btn-link dismiss-cta" -event-category="guide-cta" ga-event-action="guide-cta-dismiss">Dismiss</button>
+        </div>
+      </div>
     </div>
   </div>
 </div>
+<div class="post-bg comments"></div>
+{% include newsletter.html %}
```

Comment:
Maybe until the newsletter is perfected, since we already have a newsletter link at the footer, I would comment this out, create a JIRA card and work on the newsletter on a future PR